app/navigation/Navigator.scala

```diff
@@ -167,20 +167,20 @@ class Navigator @Inject()(appConfig: FrontendAppConfig)
     userAnswers
       .getV(PartTimePeriodsPage)
       .map { partTimePeriods =>
-        routes.PartTimeNormalHoursController.onPageLoad(previousIdx)
+        if (partTimePeriods.isDefinedAt(previousIdx)) {
+          routes.PartTimeNormalHoursController.onPageLoad(previousIdx + 1)
+        } else {
+          routes.NicCategoryController.onPageLoad()
+        }
       }
       .getOrElse(routes.PartTimePeriodsController.onPageLoad())
   }
 
   private val partTimeNormalHoursRoutes: (Int, UserAnswers) => Call = { (previousIdx, userAnswers) =>
     userAnswers
       .getV(PartTimePeriodsPage)
-      .map { partTimePeriods =>
-        if (partTimePeriods.isDefinedAt(previousIdx)) {
-          routes.PartTimeHoursController.onPageLoad(previousIdx + 1)
-        } else {
-          routes.NicCategoryController.onPageLoad()
-        }
+      .map { _ =>
+        routes.PartTimeHoursController.onPageLoad(previousIdx + 1)
```

Comment:
we don't want a +1 here

Reply:
@jordanrowe I'm being a major league derp. Fixed.